### ISSUE TITLE
unclear wording in charter about proposals being (?) specifications

### ISSUE THREAD

**Issue by dbaron** [WICG/admin#98]
The [WICG charter](https://wicg.github.io/admin/charter.html) in the [section on Community and Business Group Process and Patent Policy](https://wicg.github.io/admin/charter.html#process) contains the text:

> As with other Community Groups, W3C seeks organizational licensing commitments under the [W3C Community Contributor License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/) (Proposals in this Community Group charter are applicable "Specification" in the CLA).

It's not particularly clear what this parenthetical means.  What I *hope* it means is that any proposal (for example, an explainer) counts as a "Specification" in the terminology of the [Community and Business Group Process](https://www.w3.org/community/about/agreements/), particularly where it says:

> The label “Community Group Report” refers to any document produced by a Group. Some Community Group Reports are Specifications. The following rules apply to Specifications:
>
> * drafts are governed by Community Contributor License Agreement (CLA). W3C will provide a template for including copyright information.
> * they must include the name the group that published the deliverables and link to a public page about the group.
> * they must include a publication date.
> * the history of Contributions (as defined under the CLA) must be archived permanently on the W3C Web site.
> * They must include boilerplate language required by W3C (e.g., notice that Contributions are made under the CLA and that certain conditions apply).
> * When published on the W3C site, they must not violate the W3C Privacy Policy.
> * The content and style of the Specification must not cause confusion about its status, in particular with respect to W3C Technical Reports.

... and later:

> Other types of deliverables must be publicly available and must be archived permanently.

It would be helpful if this sentence were clearer; it's not clear what "are applicable" means in this context.

**Comment by marcoscaceres** [comment]
yeah, I'm guessing that should be "Community Group Report". We should update that.

**Comment by marcoscaceres** [comment]
Should probably also say "Community Submission", to cover the discourse threads and GitHub Issues.

**Comment by marcoscaceres** [comment]
@dbaron, how about: 

> Like other Community Groups, the WICG is subject to the W3C Community and Business Group Process and Patent Policy, which includes requirements for organizational licensing commitments under the W3C Community Contributor License Agreement (CLA). For the purposes of the CLA, proposals within the WICG will be considered "Specifications," which means they are subject to the same rules and requirements that apply to other W3C deliverables, including rules for publication, copyright, and archiving.

**Comment by dbaron** [comment]
I think this comment is from a personal perspective, since I originally raised this issue back when I was the AC Rep for a company that I no longer work for -- and I shouldn't speak on behalf of my current company on this stuff since I'm not the AC Rep.

The revision looks mostly good to me, although I'm a little concerned that the clause beginning with "which means" might be too much analysis of the policy document (which could potentially conflict with it in some way).  If you think that analysis is important, it might be better to make it more clearly advisory, such as by changing "which means" to instead start a new sentence like "Note that this means".

**Comment by marcoscaceres** [comment]
Thanks @dbaron for the feedback regardless. 

@cwilso, are you AC rep for Google? Can you take a look at the updated text at https://github.com/WICG/admin/pull/173?
